src/dom-inspector/DOMInspector.tsx

```diff
@@ -20,7 +20,7 @@ const domIterator = function* (data: any) {
       if (node.nodeType === Node.TEXT_NODE && node.textContent.trim().length === 0) continue;
 
       yield {
-        name: `${node.tagName}[${i}]`,
+        name: `${node.tagName}-${i}`,
```

Comment:
Makes it easier to parse [here](https://github.com/getsentry/react-inspector/pull/4/files#diff-6d4bb7ab44c26f9e4042df89700eb7a65dede8d3685d975502c286145a525208R47-R53)